### PR TITLE
Move some logs to `DEBUG`

### PR DESCRIPTION
**Pull request by luckysori**

### CODE REVIEW

File: daemon/src/maker_inc_connections.rs [hunk 1]
```diff
@@ -605,7 +605,7 @@ async fn upgrade(
 
     let daemon_version = daemon_version.unwrap_or_else(|| String::from("<= 0.4.7"));
 
-    tracing::info!(%taker_id, %taker_address, %negotiated_wire_version, %daemon_version, "Connection upgrade successful");
+    tracing::debug!(%taker_id, %taker_address, %negotiated_wire_version, %daemon_version, "Connection upgrade successful");
```
Comment (da-kami) [inline daemon/src/maker_inc_connections.rs]: Could we include `%taker_address` in the above `Connection is ready` message and downgrade this one to `trace`? I feel having both is somewhat redundant.

**Review by da-kami — APPROVED** [review]

File: daemon/src/maker_inc_connections.rs (hunk 1 above)
Comment (luckysori) [reply daemon/src/maker_inc_connections.rs]: I'm okay with that. See https://github.com/itchysats/itchysats/pull/1874/commits/9ddaf8281d9e6893dcebdd3ca124f319151bd740.

**Review by klochowicz — APPROVED** [review]

**Review by bonomat — APPROVED** [review]
LGTM

**Pull request merged**